Makefile

```diff
@@ -668,16 +668,11 @@ api/api.md: third_party/ build/toolchain/bin/protoc-gen-doc$(EXE_EXTENSION)
 	$(PROTOC) api/*.proto \
 		-I $(REPOSITORY_ROOT) -I $(PROTOC_INCLUDES) \
   		--doc_out=. \
-  		--doc_opt=markdown,api.md
+  		--doc_opt=markdown,api_temp.md
 # Crazy hack that insert hugo link reference to this API doc -)
```

Comment:
it seems like this may not be true anymore?

Reply:
This does look like an additional step. will reproduce and be sure. I believe this change was made and merged in previous PR. If so, I'll reopen and make another change

Reply:
Reproduced and it seems to hold true.